### DISCUSSION TITLE
Welcome to ray-optics Discussions!

### DISCUSSION THREAD

**Discussion by mjhoptics** [mjhoptics/ray-optics#28] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  Please feel free to ask questions about ray-optics, or optics in general.

**Comment by dibyendumajumdar** [comment]
Great, thanks!
